mev-boost-rs/src/service.rs

```diff
@@ -30,18 +30,19 @@ impl Default for Config {
     }
 }
 
-fn parse_url(input: &str) -> Option<Url> {
-    if input.is_empty() {
-        None
-    } else {
-        input
-            .parse()
-            .map_err(|err| {
-                tracing::warn!("error parsing relay from URL: `{err}`");
-                err
-            })
-            .ok()
+fn parse_relay_endpoints(relay_urls: &[String]) -> Vec<RelayEndpoint> {
+    let mut relays = Vec::new();
```

Comment:
nitpick but I prefer to just use the macro for this

```suggestion
    let mut relays = vec![];
```